.github/actions/test-dotnet/replace-xunit-maxParallelThreads.ps1

```diff
@@ -0,0 +1,12 @@
+param ($MaxParallelThreads)
+
+Write-Output "Replacing maxParallelThreads in xunit.runner.json files to $MaxParallelThreads."
+
+$counter = 0
+
+Get-ChildItem "*/**/xunit.runner.json" -Recurse | ForEach {
```

Comment:
`ForEach` is also an alias and this makes PSScriptAnalyzer angry. I don't think it's a significant improvement over `ForEach-Object` anyway, and it might be confusing with the `foreach` statement considering the case-insensitivity in pwsh. I've already ranted about how `%` is a must for me and that's why I added it to the whitelist, but if you don't want that then I think `ForEach-Object` would be better than going half-measures. What do you think?